docs/api_wiki/sonic_asic_methods/show_interfaces_portchannel.md

```diff
@@ -0,0 +1,47 @@
+# show_interfaces_portchannel
```

Comment:
Is this doc update related to the test? Can you update the doc in another PR if it's not related?